architecture/core_concepts/routes.adoc

```diff
@@ -332,3 +332,96 @@ certificate to validate the endpoint certificate, securing the connection
 from the router to the destination.
 
 ====
+
+== Route Hostnames
+In order for services to be exposed externally, an OpenShift route allows
+you to associate a service with an externally-reachable hostname.
+This edge hostname is then used to route traffic to the service.
+
+.A Route with a specified host:
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: host-route
+spec:
+  host: www.example.com  <1>
+  to:
+    kind: Service
+    name: service-name
+----
+
+<1> Specifies the externally-reachable hostname used to expose a service.
+====
+
+
+.A Route without a host:
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: no-route-hostname
+spec:
+  to:
+    kind: Service
+    name: service-name
+----
+
+====
+
+If a hostname is *not* provided as part of the route specification, then
+OpenShift will automatically generate one for you. The generated hostname
+is of the form `*$routename[.$namespace].$suffix*`.
+
+The following example shows the OpenShift generated hostname for the above
+configuration of a route without a host added to a namespace `my-namespace`:
+
+.Generated Hostname
+====
+
+[source,yaml]
+----
+no-route-hostname.my-namespace.router.openshift.local <1>
+----
+
+<1> The generated hostname suffix is the default routing subdomain `*router.openshift.local*`.
+====
+
+
+=== Custom default routing subdomain
+The suffix or the default routing subdomain can be tailored to your
+environment using the OpenShift master configuration.
+The following example shows how you can set the configured suffix to
```

Comment:
Since we can't assume the reader is a developer (end-user) or a cluster administrator, we may want to say "how an administrator can set the configured suffix" here, just to be extra clear (even if it's fairly obvious as is).